Sources/StreamChatUI/Common Views/ChatChannelCreateNewButton_Tests.swift

```diff
@@ -0,0 +1,80 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatChannelCreateNewButton_Tests: XCTestCase {
+    let containerView: UIView = {
+        let view = UIView(frame: .init(x: 0, y: 0, width: 44, height: 44))
+        view.backgroundColor = UIColor.white
+        return view
+    }()
+    
+    let defaultSize = CGSize(width: 44, height: 44)
+    
+    func test_defaultAppearance() {
+        let view = ChatChannelCreateNewButton().withoutAutoresizingMaskConstraints
+        containerView.addSubview(view)
+        AssertSnapshot(containerView, variants: [.small], size: defaultSize)
+    }
```

Comment:
look at the test for avatar view, when the element doesn't have a fixed size, you can just add sizing constraints to make it resize properly. I don't think we should use the `size` parameter here.

Reply:
Isn't the same thing? I think what we are doing in the avatar view is what "size" argument is doing, shouldn't we just use this instead? Since it is already available for us 🤔